### PR TITLE
add new sensor Sparkfun Micropressure or Honeywell MPRLS0025PA00001A

### PR DESCRIPTION
**Pull request by MartinVerges**
## Description:

This adds the Sparkfun Micropressure or Honeywell MPRLS0025PA00001A sensor tested on my esp32.
As I'm new to tasmota, platformio and c/c++ development, please check my code before merging.

## Checklist:
  - [x] The pull request is done against the latest development branch
  - [x] Only relevant files were touched
  - [x] Only one feature/fix was added per PR and the code change compiles without warnings
  - [ ] The code change is tested and works with Tasmota core ESP8266 V.2.7.4.9
  - [x] The code change is tested and works with Tasmota core ESP32 V.2.0.1
  - [X] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

### CODE REVIEW

File: tasmota/xdrv_01_webserver.ino [hunk 1]
```diff
@@ -921,6 +921,10 @@ void WSContentSend_Temp(const char *types, float f_temperature) {
   WSContentSend_PD(HTTP_SNS_F_TEMP, types, Settings->flag2.temperature_resolution, &f_temperature, TempUnit());
 }
 
+void WSContentSend_Pressure(const char *types, float f_pressure) {
+  WSContentSend_PD(HTTP_SNS_F_PRESSURE, types, Settings->flag2.pressure_resolution, &f_pressure);
+}
+
```
Comment (arendst) [inline tasmota/xdrv_01_webserver.ino:927]: Don't add it here. Move it into your driver. As there are several other pressure drivers look how they do it.

Also have a look at their "SeaPressure" functionality. A free bonus if you manage to integrate your driver like the other ondes did.

File: tasmota/xsns_95_mprls0025pa00001a.ino [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+#endif  // USE_WEBSERVER
+
+bool Xsns95(uint8_t function) {
+  if (!I2cEnabled(XI2C_65)) { return false; }
+
+  bool result = false;
+
+  switch (function) {
+    case FUNC_INIT:
+      AddLog(LOG_LEVEL_INFO, PSTR(D_LOG_I2C "MPR: FUNC_INIT"));
```
Comment (arendst) [inline tasmota/xsns_95_mprls0025pa00001a.ino:60]: This construction also is not correct. It forces function calls to service while most of the time your drivers hardware will not be present.

Again, have a look at other I2C drivers how they manage the lack of hardware by disabling all function calls except the ``FUNC_INIT``.

**Comment by MartinVerges** [comment]
I tried to change what you mentioned, please review and let me know.

**Comment by MartinVerges** [comment]
can you merge it?

**Review by ascillato — APPROVED** [review]

**Comment by MartinVerges** [comment]
Still waiting for it what can I do to get it merged?

**Comment by MartinVerges** [comment]
Is anything wrong, is there's something i need to do or change?

**Comment by ascillato2** [comment]
bump

**Pull request closed without merging**